corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -809,6 +824,11 @@ def _report_columns(self):
         return []
 
     @property
+    @memoized
+    def _number_columns(self):
+        return [col["field"] for col in self._report_columns if col["aggregation"] in ["avg", "sum"]]
```

Comment:
made the expression that finds columns that must be of number type a property since i had to remove it from the function that builds the indicators